app/components/inputs/Input.js

```diff
@@ -97,6 +98,13 @@ class Input extends React.Component{
               ) : null}
             </div>
           ) : null}
+          {!invalid && value && validMessage ? (
+            <div className="input-valid-message-area">
+              <div>
```

Comment:
No need for this extra `div` here. `{validMessage}` can wrap itself however it wants and the previous `<div className="input-valid-message-area">` already provides separation.